src/table.ts

```diff
@@ -764,15 +769,15 @@ class Table {
    * @param {function} callback The callback function.
    */
   private _mutate(
-      method: string, rows: object|object[], callback: s.CommitCallback): void {
-    this.database.runTransaction((err, transaction) => {
+      method: string, rows: object|object[], callback: CommitCallback): void {
+    this.database.runTransaction((err: Error|null, transaction?: Transaction|null) => {
       if (err) {
         callback(err);
         return;
       }
 
-      transaction[method](this.name, rows);
-      transaction.commit(callback);
+      transaction![method](this.name, rows);
```

Comment:
@callmehiphop Wanted to mention that, when enabling noImplicitAny, an error pops up here saying `Element implicitly has an 'any' type because type 'Transaction' has no index signature.`

Doing something like this works but isn't a good fix:
```
  export class Transaction extends Dml {
+  [k: string]: any;
   commitTimestamp?: PreciseDate;
```

Don't know how you guys want to go about solving this one. This same error pops up in multiple different places in other files including `src/v1/spanner_client.d.ts`